src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js

```diff
@@ -0,0 +1,78 @@
+import {
+  FETCH_PERSONAL_INFO,
+  FETCH_PERSONAL_INFO_FAILED,
+  FETCH_PERSONAL_INFO_SUCCESS,
+  UPDATE_PENDING_VERIFICATIONS,
+  UPDATE_VERIFICATIONS,
+} from '../actions';
+
+const initialState = {
+  personalInfo: null,
+  isLoading: false,
+  error: null,
+};
+
+const defaultUserInfo = {
+  'vye::UserInfo': {
+    pendingVerifications: [],
+    verifications: [],
+  },
+};
+
+const personalInfo = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_PERSONAL_INFO:
+      return {
+        ...state,
+        isLoading: true,
+        error: null,
+      };
+    case FETCH_PERSONAL_INFO_SUCCESS:
+      return {
+        ...state,
+        personalInfo: action.response,
+        isLoading: false,
+      };
+    case UPDATE_PENDING_VERIFICATIONS:
```

Comment:
This action is flagged as not found in the actions file. Is this case still needed?

Reply:
unit testing will be created once we show the address and the direct deposit in the UI in the profile page.